settings.gradle.kts

```diff
@@ -16,6 +16,7 @@ dependencyResolutionManagement {
 rootProject.name = "kotlin-inject"
 
 include(":kotlin-inject-runtime")
+include(":kotlin-inject-kmp-runtime")
```

Comment:
I think `kotlin-inject-runtime-kmp` would match the ordering like `kotlin-inject-compiler-ksp` a bit better.